src/nested_pandas/nestedframe/__init__.py

```diff
@@ -0,0 +1 @@
+from .core import *  # noqa
```

Comment:
Do we really need star-import here? It would add `pd` and other stuff we don't really need